roles/base/templates/unattended_upgrades.j2

```diff
@@ -0,0 +1,55 @@
+// Automatically upgrade packages from these (origin:archive) pairs
+Unattended-Upgrade::Allowed-Origins {
+        "${distro_id}:${distro_codename}-security";
```

Comment:
I copied this from Ubuntu, does this work for Debian as well?

Reply:
Maybe base your changes on the file installed by the Debian package?